rest_framework-stubs/fields.pyi

```diff
@@ -79,6 +85,8 @@ _RP = TypeVar("_RP")  # Representation Type
 class SupportsToPython(Protocol):
     def to_python(self, value: Any) -> Any: ...
 
+_DefaultInitial = Union[_VT, Callable[[], _VT], None, _Empty]
```

Comment:
Empty is not something the user is expected to pass externally, but is a fallback valiue user internally by the init method. As such this is wrong to put it inside the type alias here.

Reply:
If user won't pass it as argument it doesn't mean that it shouldn't be included into type signature, type signature **must** include all valid values, example where that useful:

```
def __init__(self, ..., default: _DefaultInitial,...):
    if default is None or iscallable(default):
      ...
    else:
      ... <- here default will have type Union[_VT, _Empty], otherwise user would think that he checked everything and default is _VT

Reply:
Maybe, but DRF doesn't expect user to pass it in. This is a non standard edge case.

Reply:
how it's edge case? if you call serializer without default argument in init you will have default=empty, even if it edge case I see zero reason why empty shouldn't be in the union